src/internal/m365/collection/groups/handlers.go

```diff
@@ -43,11 +51,19 @@ type backupHandler interface {
 		folders *path.Builder,
 		tenantID string,
 	) (path.Path, error)
+
+	locationPath(c C) *path.Builder
+
+	// canMakeDeltaQueries evaluates whether the container can support a
+	// delta query when enumerating its items.
+	canMakeDeltaQueries(C) bool
```

Comment:
nit: for consistency

```suggestion
	canMakeDeltaQueries(c C) bool
```